Automattic/jetpack
# API: Adds GET roles endpoint

Syncs the list roles endpoints from WPCOM. Returns an array of roles objects that a site supports.

To test:
- Checkout `add/list-roles`
- Make a request to `/sites/$site/roles`
- Ensure that an array of role objects is return
